### ISSUE TITLE
adjust position of text and graphic labels on game setup menu

### ISSUE THREAD

**Issue by andy5995** [ZetaGlest/zetaglest-source#58]
andrius2102 has offered to do this after #37 is completed.

Most of what needs to be done will be obvious. Anything else will be caught during the review process.

![screenshot_2018-02-11_17-33-55](https://user-images.githubusercontent.com/16764864/36143919-17e23d2a-1072-11e8-81e8-aa33dd0d61ca.png)

This is only a small part of the code that would need editing and possibly refactoring to make re-positioning text easier in the future:

https://github.com/ZetaGlest/zetaglest-source/blob/d67e8c785a5c10843f3c76efb049d0381f73bffb/source/glest_game/menu/menu_state_connected_game.cpp#L205-L220

https://github.com/ZetaGlest/zetaglest-source/blob/d67e8c785a5c10843f3c76efb049d0381f73bffb/source/glest_game/menu/menu_state_custom_game.cpp#L248-L261

@Jammyjamjamman thought it might be a good idea to put scroll bars around the list of players.

Some examples of where a scroll bar is already used is at

https://github.com/ZetaGlest/zetaglest-source/blob/d67e8c785a5c10843f3c76efb049d0381f73bffb/source/glest_game/menu/menu_state_masterserver.cpp#L89-L102

andrius, don't feel a need to do that unless you really want to. We can make a separate ticket for it.

As for the positioning, feel free to re-organize the existing layout if you think of something better.

**Comment by andy5995** [comment]
Is anyone working on this? @Joshei , would you be interested in working on this?

**Comment by Joshei** [comment]
Yes, sure thank you!

**Comment by andy5995** [comment]
Make sure your repo is updated with the latest source code. I made some changes to these files recently, so if you don't update first, you're likely to get merge conflicts later.

**Comment by Joshei** [comment]
Just looked at the assignment.  Looks like a good one!  Now, how did this get so awful?  Especially the Advanced Options.  I think I can design this okay, though.  I have one outstanding question, however.  Where do we want to put the Mini Map?  Should it belong in the lower left hand corner?  Putting it in the upper left corner might be a better shared theme, however if unnecessary it would save a lot of time to not put it there!  Also, perhaps more importantly, are we going are own way with this (without comparing to MegaGlest?)

**Comment by Joshei** [comment]
Oh, I'm understanding better now.  The real image can be made to look like the one above except for the minimap...  The real image is much worse.

**Comment by andy5995** [comment]
>Also, perhaps more importantly, are we going are own way with this (without comparing to MegaGlest?)

Yes, we can go our own way.

>Where do we want to put the Mini Map? Should it belong in the lower left hand corner?

I think it can just be lowered a little, below the "10".

>Now, how did this get so awful? 

When I added 2 slots for enhanced observer mode. Everything needed to be lowered, and I didn't get to all of it.

**Comment by mathusummut** [comment]
[This is now fixed.](https://github.com/ZetaGlest/zetaglest-source/commit/2e9623c8da535b14c4d65e79fe7eb9fb6045f3d7)
